docs/js_binary.md

```diff
@@ -60,6 +61,9 @@ This rules requires that Bazel was run with
 | <a id="js_binary-entry_point"></a>entry_point |  The main script which is evaluated by node.js.<br><br>        This is the module referenced by the <code>require.main</code> property in the runtime.<br><br>        This must be a target that provides a single file or a <code>DirectoryPathInfo</code>         from <code>@aspect_bazel_lib//lib::directory_path.bzl</code>.<br><br>        See https://github.com/aspect-build/bazel-lib/blob/main/docs/directory_path.md         for more info on creating a target that provides a <code>DirectoryPathInfo</code>.   | <a href="https://bazel.build/docs/build-ref.html#labels">Label</a> | required |  |
 | <a id="js_binary-env"></a>env |  Environment variables of the action.<br><br>        Subject to <code>$(location)</code> and make variable expansion.   | <a href="https://bazel.build/docs/skylark/lib/dict.html">Dictionary: String -> String</a> | optional | {} |
 | <a id="js_binary-expected_exit_code"></a>expected_exit_code |  The expected exit code.<br><br>        Can be used to write tests that are expected to fail.   | Integer | optional | 0 |
+| <a id="js_binary-include_npm_deps"></a>include_npm_deps |  When True, npm_linked_deps from JsInfo and transitive_files in NpmLinkedPackageInfo and NpmPackageStoreInfo         providers in data targets are included in the runfiles of the target.   | Boolean | optional | True |
+| <a id="js_binary-include_transitive_declarations"></a>include_transitive_declarations |  When True, transitive_declarations from JsInfo providers in data targets are included in the runfiles of the target.<br><br>        Defaults to false since they are generally not needed at runtime and introducing them could slow down developer round trip         time due to having to generate typings on source file changes.   | Boolean | optional | False |
```

Comment:
> Defaults to false since they are

What is "they"? The providers? The data targets?

Reply:
declarations. I updated the docstring to be more specific.